### DISCUSSION TITLE
Horario preguntas recuperativas y convivencia

### DISCUSSION THREAD

**Discussion by cbuerger02** [IIC2133-PUC/2024-1#198] [Ayuda] [answered]
Hola, 

tengo una duda: ¿cuál es el horario programado para la instancia de preguntas recuperativas? lo otro, no me queda claro si la convivencia será en este mismo horario o después.

Gracias de antemano!

Saludos,
Christoph

**Comment by CataAlamos** [comment, marked as answer]
Hola! La convivencia es post recuperativa y mandamos un email a todos los que tienen justificada interrogaciones con los detalles.

**Reply by cbuerger02** [reply]
Gracias por tu respuesta, preguntaba también porque tengo un compromiso a las 12:20 en la universidad, entonces no sé si está estipulado terminar antes, para ver si alcanzo a llegar al compromiso.
